feature-libs/checkout/components/components/shipping-address/shipping-address.component.ts

```diff
@@ -197,6 +197,9 @@ export class ShippingAddressComponent implements OnInit, OnDestroy {
       ],
       actions: [{ name: textShipToThisAddress, event: 'send' }],
       header: selected && selected.id === address.id ? textSelected : '',
+      label: address.defaultAddress
```

Comment:
Should be transferred

Reply:
don't worry! as explained here https://sap-cx.slack.com/archives/D02A0NHD3C0/p1643298314004500?thread_ts=1643268177.004900&cid=D02A0NHD3C0